WasabiWords.lua

```diff
@@ -589,6 +589,122 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
+        stats.set_int("MPX_ARN_VEH_ZR3803", 500)
+        stats.set_int("MPX_ARN_VEH_DEATHBIKE", 500)
+        stats.set_int("MPX_ARN_VEH_DEATHBIKE2", 400)
+        stats.set_int("MPX_ARN_VEH_DEATHBIKE3", 400)
+        stats.set_int("MPX_AWD_WATCH_YOUR_STEP", 20)
+        stats.set_int("MPX_AWD_TOWER_OFFENSE", 50)
+        stats.set_int("MPX_AWD_THROUGH_A_LENS", 60)
+        stats.set_int("MPX_AWD_SPINNER", 80)
+        stats.set_int("MPX_AWD_YOUMEANBOOBYTRAPS", 25)
+        stats.set_int("MPX_AWD_MASTER_BANDITO", 20)
+        stats.set_int("MPX_AWD_SITTING_DUCK", 30)
+        stats.set_int("MPX_AWD_CROWDPARTICIPATION", 80)
+        stats.set_int("MPX_AWD_KILL_OR_BE_KILLED", 60)
+        stats.set_int("MPX_AWD_MASSIVE_SHUNT", 60)
+        stats.set_int("MPX_AWD_YOURE_OUTTA_HERE", 220)
+        stats.set_int("MPX_AWD_ARENA_WAGEWORKER", 6785341)
+        stats.set_int("MPX_AWD_TIME_SERVED", 474020)
+        stats.set_int("MPX_AWD_CAREER_WINNER", 36340)
+        stats.set_int("MPX_ARN_SPECTATOR_DRONE", 60)
+        stats.set_int("MPX_ARN_SPECTATOR_CAMS", 60)
+        stats.set_int("MPX_ARN_SMOKE", 50)
+        stats.set_int("MPX_ARN_DRINK", 65)
+        stats.set_int("MPX_ARN_SPEC_BOX_TIME_MS", 10800000)
+        stats.set_int("MPX_AWD_TOP_SCORE", 1045020)
+        stats.set_int("MPX_AWD_READY_FOR_WAR", 50)
+        stats.set_int("MPX_AWD_WEVE_GOT_ONE", 60)
+        stats.set_int("MPX_ARN_W_THEME_SCIFI", 10)
+        stats.set_int("MPX_ARN_W_THEME_APOC", 10)
+        stats.set_int("MPX_ARN_W_THEME_CONS", 10)
+        stats.set_int("MPX_ARN_BS_TRINKET_SAVED", -1) -- Unlock trinket and stickers.
+        stats.set_bool("MPX_AWD_UNSTOPPABLE", true)
+        stats.set_bool("MPX_AWD_CONTACT_SPORT", true)
+        stats.set_bool("MPX_AWD_BEGINNER", true)
+        stats.set_bool("MPX_AWD_FIELD_FILLER", true)
+        stats.set_bool("MPX_AWD_ARMCHAIR_RACER", true)
+        stats.set_bool("MPX_AWD_LEARNER", true)
+        stats.set_bool("MPX_AWD_SUNDAY_DRIVER", true)
+        stats.set_bool("MPX_AWD_THE_ROOKIE", true)
+        stats.set_bool("MPX_AWD_BUMP_AND_RUN", true)
+        stats.set_bool("MPX_AWD_GEAR_HEAD", true)
+        stats.set_bool("MPX_AWD_DOOR_SLAMMER", true)
+        stats.set_bool("MPX_AWD_HOT_LAP", true)
+        stats.set_bool("MPX_AWD_ARENA_AMATEUR", true)
+        stats.set_bool("MPX_AWD_PAINT_TRADER", true)
+        stats.set_bool("MPX_AWD_SHUNTER", true)
+        stats.set_bool("MPX_AWD_JOCK", true)
+        stats.set_bool("MPX_AWD_WARRIOR", true)
+        stats.set_bool("MPX_AWD_T_BONE", true)
+        stats.set_bool("MPX_AWD_MAYHEM", true)
+        stats.set_bool("MPX_AWD_WRECKER", true)
+        stats.set_bool("MPX_AWD_CRASH_COURSE", true)
+        stats.set_bool("MPX_AWD_ARENA_LEGEND", true)
+        stats.set_bool("MPX_AWD_PEGASUS", true)
+        stats.set_packed_stat_bool(25010, true) -- Skip arena wall help.
+        stats.set_packed_stat_bool(25014, true) -- Skip arena wall tutorial.
```

Comment:
This isn't unlocking anything that I can see in the scripts.

Reply:
Check `am_mp_arena_garage`, they are used to determine if you've seen the Arena War Wall help or tutorial before.